File: messageGo/grpc/pubsubEventData/slot_level_upgrade.go
package pubsubEventData

type SlotLevelUpgradeEvent struct {
	MsgVersion int64 `json:"msgVersion"` // 消息版本号 值为毫秒时间戳
	UserId     int64 `json:"userId"`
	SlotPos    int32 `json:"slotPos"`
	Level      int32 `json:"level"`
}

File: messageDotNet/grpc/EventData/SlotLevelUpgrade.cs
using System;
using UnityEngine;

[Serializable]
public class SlotLevelUpgradeEvent
{ // 消息版本号 值为毫秒时间戳
    public long MsgVersion;
    public long UserId;
    public int SlotPos;
    public int Level;

    public string ToJson()
    {
        return JsonUtility.ToJson(this);
    }

}